src/js/components/molecules/auto-complete/auto-complete.scss

```diff
@@ -30,7 +30,10 @@
     }
 
     &__list-item {
-        @extend .copy--small;
+        font-size: 14px;
+        line-height: 1.5;
+        margin-top: -2px;
+        margin-bottom: 23px;
```

Comment:
And make this `margin-bottom: 21px;`

Reply:
i started to look into it and found yet another rabbit hole. i couldn't find auto-complete in the site, so i looked at business-card. some other style sets margin-top to 0 negating this setting, which overrides a -8px, which overrides yet another setting from another class ...

i think we should initially replace these extends and then go over the components and correct them.